.github/workflows/tests.yml

```diff
@@ -22,12 +22,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        include:
-          - node-version: 12.x
-            lint: true # Linter is run only once to shorten the total build time
```

Comment:
Reasoning for removing this logic:

👎 Will now take 2-5s longer per job

👍 In comparison, running tests takes over 3m per job
👍 Slightly increase confidence
👍👍👍 Reduce complexity